config/dynamic.c

```diff
@@ -295,6 +295,8 @@ cleanup_dy(struct dynamic_config *dy_conf)
 
 const struct luaL_reg dylib_lua_c_funcs [] = {
 	{"update_gt_lua_states", l_update_gt_lua_states},
+	{"l_update_gt_lua_states_incrementally",
```

Comment:
Replace the Lua name `"l_update_gt_lua_states_incrementally"` with `"internal_update_gt_lua_states_incrementally"` to avoid oveloading the meaning of the prefix `l_`, which we are only using for C functions that meet the Lua prototype.